tools/svaba.cwl

```diff
@@ -52,7 +52,7 @@ inputs:
         items: File
         inputBinding:
           prefix: '--normal-bam'
-    secondaryFiles: [{pattern: '^.bai', required: true}]
+    secondaryFiles: [{pattern: '^.bai', required: false}, {pattern: '.bai', required: false}, {pattern: '^.crai', required: false}, {pattern: '.crai', required: false}]
```

Comment:
Hmmm, it'd be nice to give sort of the "this" or "that", like must have one of. Probably wishful thinking

Reply:
🔥